17-01-24/B/React-with-Arrays/react-with-arrays_fruits/src/components/Card.js

```diff
@@ -1,5 +1,10 @@
 import "./Card.css";
 
-export default function Card({ name }) {
-  return <p className="card">{name}</p>;
+export default function Card({ name, color }) {
+  // return <p className="card">{name}</p>;
+  return (
```

Comment:
The inline style coding should work, but we should refrain from using it and add the style to the css file. You could use the following instead:

`return <p className={`card card--${color}`}>{name}</p>;`